### ISSUE TITLE
behavioral norms

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/PWETF#8]
A colleague suggested composing list of behavioral norms to help guide members about best practices in settings such as GitHub, F2F meetings, IRC, email, and phone calls. 

Some tips might include 
* limiti the number of GitHub or email responses within a certain time frame
* request permission to send private messages
* follow queue management rules in IRC to help meetings run smoothly

Setting some rules might help avoid some behaviors that feel like bullying even if they are not intended as such.

**Comment by tripu** [comment]
In principle, I'm not convinced of the benefits of imposing this kind of constraints (except for the third example in the list, but that is just IRC etiquette and is routinely enforced by chairs anyway).

Threads (by mail or on GH) are recycled and reopened quite often, even after months of total inactivity, for good reasons. One-to-one communication, regardless of the medium used, is very valuable for bona fide digressions, questions, follow-ups and personal off-topics, and to reduce noise in public channels.

If one person feels overwhelmed by the cognitive overload of attending many notifications, alerts or messages, they can explain that to the group and excuse themselves from some of those channels/threads (and I think the group should accommodate that, to the extent that it's reasonable). Similarly, if someone doesn't tolerate well social interaction outside structured meetings, they can inform the rest, and ask that they be contacted in certain ways (and again, I would expect others to be respectful of that request).

Adding more layers of processes or rules that limit the majority to prevent potential issues that may affect (I presume) a small minority of individuals and situations&nbsp;&mdash;&nbsp;I fear that may be counterproductive.

(IMHO, where H stands for &ldquo;humble&rdquo; :)

**Comment by nigelmegitt** [comment]
#23 is an example where at least some definition of behavioural norms would be helpful.

**Comment by RachelComerford** [comment]
I think this is an opportunity to introduce international business etiquette to those of use that have been working on a more national basis - this would include (for example) include information about how business cards are appropriately exchanged as well as cultural expectations in terms of language. It's a chance to enable understanding (I know why this person did this) instead of setting constraints.

**Comment by vlevantovsky** [comment]
I think the concept of behavioral norms is useful and should be addressed; however, as with anything else in life, "the devil is in the details", and trying to regulate every aspect of behavior can easily be counterproductive - e.g. 'heated' email exchanges can be very beneficial and can help solve problems quickly.
As far as behavioral norms in general (and business etiquette in particular) are concerned, I do agree with @RachelComerford, in fact the [draft CEPC](https://github.com/w3c/PWETF/files/2555145/DraftCEPCforPWETF.v8.pdf) I wrote last year has a section entitled "Community Conduct Guidelines" that aims to do exactly that.

**Comment by vlevantovsky** [comment]
I think we can close this issue without any further actions. Many things have changed since the issue was opened, we have a whole new "Expected Behaviors" section added where we cover most important aspects of interactions and behavioral norms.

**Comment by nigelmegitt** [comment]
I disagree that the expected behaviours section covers the kinds of behavioural norms that are listed in this issue. However I do agree that a) they are needed and b) they don't need to be in the CEPC. 

Rather, the CEPC sets some overriding "principles" and the Guide would be a better place to explain "here are the behavioural norms we have established to make it easier for us to work within the CEPC" with the understanding being that if a person strays outside those, they need to be extra careful.
